acceptance_tests/features/Exception_manager.feature

```diff
@@ -1,6 +1,6 @@
 Feature: Check exception manager is called for every topic and handles them as expected
 
-  Scenario: Bad Json Msg sent to every topic, msg arrives in exception manager
+  Scenario: A Bad Json Msg sent to every topic, msg arrives in exception manager
```

Comment:
This seems like a good short term experiment/improvement, but it's going to be prone to changing and forgetting why/how this scenario was run first. 

Once we've merged this and seen if we have an improvement in test reliability, perhaps we should have cards on the board to add a pre-run hook that performs similar actions to warm up PubSub, rather than relying on this scenario running first.

Reply:
I did initially look at having something that ran before all the scenarios to get pubsub active, but it would either involve putting legit messages on to the queues which would require surveys and such being set up, or putting dummy messages which would then cause errors in the services, so using the scenario that looks for errors anyway seemed to be a sensible choice.

Reply:
Yes, I think it makes good sense as a short term solution, but it won't be reliable long term as we could easily accidentally put another scenario in front of it. Perhaps, since we're relying on the order for now, we should put `1` at the front of the scenario along with a short comment explaining why, just so we don't accidentally supplant it.

Reply:
Yea thats fair, I'll add a comment in giving a short background